contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -568,22 +570,30 @@ def requisites(self, request, pk=None):
             ),
         )
 
-    @detail_route(methods=["get"])
+    @action(detail=True, methods=["get"])
```

Comment:
Updated as I know `default_route` is deprecated.

Reply:
Just trying to give me merge conflicts in my Django 3 PR!